zerver/lib/bugdown/__init__.py

```diff
@@ -1173,6 +1174,15 @@ def run(self, lines):
                 if re.search(match_re, content):
                     current_message.alert_words.add(word)
 
+            for command in slash_commands:
```

Comment:
Should this be part of the `AlertWordsNotificationProcessor` or should there be a separate preprocessor for slash commands?